### DISCUSSION TITLE
REOPEN: rAudio 20211009 - Swiping gone?

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#550] [Fixed Issues]
At least on Android devices swiping for changing windows (e.g. from playback to library screen) is no more possible.

**Comment by rern** [comment]
Please try the UPDATE branch.

**Reply by klassikmann** [reply]
> Please try the UPDATE branch.
Looks good.

**Reply by klassikmann** [reply]
rAudio 20211010: Fixed

**Comment by klassikmann** [comment]
rAudio 20211010: 
Now swiping gone on the Win10 using Firefox and Chrome. Android Firefox o.k.

**Comment by rern** [comment]
Now swipe is enable only if a touch device, by `navigator.maxTouchPoints`, was detected.

**Reply by klassikmann** [reply]
@rern 
Is there a constraint not to allow mouse-based swiping on non-touch devices? 
I'm using rAudio on several Android devices and - at the same time - on my desktop PC. Now I have to enable "Top-Bottom bars" for simultaneous usage which I have been able to avoid it so far.

**Reply by cmh714** [reply]
Actually you dont have to enable the Bars......when its disabled, the bottom bar is actually "invisible" but not gone like the top bar. A very very nice bug :)

**Reply by klassikmann** [reply]
Nice!
But nevertheless I'm curious to hear about possible technical reasons to restrict swiping, even mouse-based one, to non-touch devices.

**Reply by rern** [reply]
Transparent bottom bar is not a bug. It's been there for those who are hesitant of hidden bars or have trouble swiping. While bars are hidden, click/tap guide (top-center of coverart) shows where it is.

![12](https://user-images.githubusercontent.com/10094236/137059038-909d3682-35a3-4882-abd9-768802dc275b.jpg)

Swipe used to rely on 3rd party plugin which caused issues. Now it runs with native code which based on touch events only. Mouse events, which caused issues if combined, were omitted.

**Reply by klassikmann** [reply]
@rern 
Thank you for the explanation.
